src/main/java/seedu/address/logic/commands/CountryNoteCommand.java

```diff
@@ -0,0 +1,47 @@
+package seedu.address.logic.commands;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.country.Country;
+import seedu.address.model.note.Note;
+
+public class CountryNoteCommand extends Command {
```

Comment:
javadoc comment

Reply:
Hmm strange that checkstyle didn't throw a warning. Added.